src/views/addItemView/index.tsx

```diff
@@ -18,6 +18,14 @@ export default function AddItemView() {
   const handleSubmit = async (e: React.FormEvent) => {
     e.preventDefault();
 
+    const { high, low } = itemFormData;
+
+    // Check if high is greater than low
+    if (high <= low) {
```

Comment:
```suggestion
    if (high < low) {
```

Reply:
high === low is acceptable in the case that they only have a different price for a new vs used item